src/components/ebay-ccd/index.marko

```diff
@@ -0,0 +1,61 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+import type { WithNormalizedProps } from "../../global";
+
+static interface CCDInput extends Omit<Marko.Input<"div">, `on${string}`> {
+    max?: string;
+    min?: string;
+    "charger-icon"?: string;
```

Comment:
This should be an enum, like `"none" | "included" | "notIncluded"`

Reply:
Also, I think we generally use kebab-case for attributes like this. Is `"notIncluded"` standard terminology or was it chosen arbitrarily?